src/status_im/contexts/wallet/common/sheets/account_origin/view.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.contexts.wallet.common.sheets.account-origin.view
+  (:require
+    [quo.core :as quo]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.constants :as const]
+    [status-im.contexts.wallet.common.sheets.account-origin.style :as style]
+    [utils.i18n :as i18n]))
+
+(defn- header
+  [text]
+  [quo/text
+   {:weight :semi-bold
+    :style  style/header-container}
+   text])
+
+(defn- description
+  [text]
+  [quo/text
+   {:size  :paragraph-2
+    :style style/desc-container}
+   text])
+
+(defn- section
+  [title desc]
+  [:<>
+   [header title]
+   [description desc]])
+
+(defn- view-internal
+  []
+  [:<>
```

Comment:
Isn't it better to wrap this in a `view` to make sure we are rendering everything as expected?

If this component is being used inside a 
`[rn/view {:style {:flex-direction :row}}}]` 
will be showed as expected?